dvc.lock

```diff
@@ -865,51 +851,150 @@ stages:
+          min_n_sales: 30
           num_quantile:
           - 3
           - 5
           - 10
+          geographies:
+          - meta_township_code
+          - meta_nbhd_code
+          - loc_tax_municipality_name
+          - loc_ward_num
+          - loc_census_puma_geoid
+          - loc_census_tract_geoid
+          - loc_school_elementary_district_geoid
+          - loc_school_secondary_district_geoid
+          - loc_school_unified_district_geoid
+  upload:
+    cmd: Rscript pipeline/06-upload.R
+    deps:
+    - path: output/assessment_card/model_assessment_card.parquet
+      hash: md5
+      md5: 3442b0b0fb25364caba810a507213109
+      size: 38822670
+    - path: output/assessment_pin/model_assessment_pin.parquet
+      hash: md5
+      md5: ae6242ed4427ccd87acab2d87435ab8f
+      size: 41641680
+    - path: output/feature_importance/model_feature_importance.parquet
+      hash: md5
+      md5: 61db6f11d2ea7aa53d6990445b5d9cd2
+      size: 8582
+    - path: output/metadata/model_metadata.parquet
+      hash: md5
+      md5: 5bfe8e50f50463253a3f8f4fa3164bb8
+      size: 29757
+    - path: output/parameter_final/model_parameter_final.parquet
+      hash: md5
+      md5: b234a91486b487642e8738306f87c25c
+      size: 8857
+    - path: output/parameter_range/model_parameter_range.parquet
+      hash: md5
+      md5: 150000269b5873fa1b3eaeeff7887ce2
+      size: 501
+    - path: output/parameter_search/model_parameter_search.parquet
+      hash: md5
+      md5: 150000269b5873fa1b3eaeeff7887ce2
+      size: 501
+    - path: output/performance/model_performance_assessment.parquet
+      hash: md5
+      md5: 6c43dfc44d5e8186f037b5c6d7bbd8b1
+      size: 573773
+    - path: output/performance/model_performance_test.parquet
+      hash: md5
+      md5: 9867d9222eb5ff618f69b185ffc7452c
+      size: 1060602
+    - path: output/performance_quantile/model_performance_quantile_assessment.parquet
+      hash: md5
+      md5: 8fb50ba32609879ad5fc9b196e07bdae
+      size: 461742
+    - path: output/performance_quantile/model_performance_quantile_test.parquet
+      hash: md5
+      md5: 5d5b3e0c69fab782974f89c4bbbf75fb
+      size: 1055715
+    - path: output/shap/model_shap.parquet
+      hash: md5
+      md5: 150000269b5873fa1b3eaeeff7887ce2
+      size: 501
+    - path: output/test_card/model_test_card.parquet
+      hash: md5
+      md5: e95956454d04a68669f04f5355af3b5e
+      size: 1342825
+    - path: output/timing/model_timing.parquet
+      hash: md5
+      md5: 736810f7363817b6023d98b1e74d05af
+      size: 6032
+    - path: output/workflow/fit/model_workflow_fit.zip
+      hash: md5
+      md5: 5a607521588c3aca5761150390082127
+      size: 15244546
+    - path: output/workflow/recipe/model_workflow_recipe.rds
+      hash: md5
+      md5: c672f98b0b68e5a16adb0b687b43adca
+      size: 4199953
+    - path: reports/performance/performance.html
+      hash: md5
+      md5: 004b653e50e9513fc04ad1fc1d5ca544
+      size: 80
```

Comment:
**[Question, non-blocking]** Looks like [we first added this stage in December](https://github.com/ccao-data/model-condo-avm/commit/9acbc3340eb65598bb81e13707f4325711886c29). Any idea why DVC is only now adding it to the lockfile?

Reply:
I think we basically just didn't run the condo model locally to update the `dvc.lock` file, it only ran on Batch.